src/unixsocket_http/data.clj

```diff
@@ -25,20 +25,27 @@
 
 ;; ## Request
 
+(defn- add-query-params!
+  [^HttpUrl$Builder builder {:keys [query-params]}]
+  (doseq [[k value-or-collection] query-params
+          v (if (coll? value-or-collection)
```

Comment:
@xsc Would'nt `(sequential? ...)` be better than `(coll? ...)` here? a map is a coll too.

Reply:
I thought about this as well. I opted in favour of `coll?` because `(sequential? #{})` evaluates to `false` and I didn't want to exclude sets.